health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/InboxFilter.js

```
@@ -0,0 +1,116 @@
+  //   params,
+  //   config: {},
+  // })?.data?.Project;
+
+  // TODO : Need to confirm whether to show all the campaigntypes by default or get the unique campaigntypes from response and show them
+  // const campaignTypeOptions = [...new Map(result.map((item) => [item.name, { code: item.name, name: item.name }])).values()];
+  // TODO : Need to confirm whether to show all the boundaryCodes by default or get the unique boundaryCodes from response and show them
+  // const boundaryTypeOptions = [
+  //   ...new Map(result.map((item) => [item.address?.boundary, { code: item.address?.boundary, name: item.address?.boundary }])).values(),
+  // ];
+
+  let options = [
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'options' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)